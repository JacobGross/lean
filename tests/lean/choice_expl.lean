namespace N1
  definition pr {A : Type} (a b : A) := a
end N1

namespace N2
  definition pr {A : Type} (a b : A) := b
end N2

open N1 N2
constant N : Type.{1}
constants a b : N
check @pr
check @pr N a b
check pr a b
