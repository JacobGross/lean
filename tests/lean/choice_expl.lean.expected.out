pr : Π {A : Type}, A → A → A
pr a b : N
pr a b : N
